docs/CodingStyleGuidelines.md

```diff
@@ -736,28 +736,28 @@ The above method for passing virtual interfaces to the verification environment
 
 4.4.3. Components only get handles to virtual interfaces from the UVM resource database.
 
-4.4.4. The component defines a string configuration field that is used to look up the virtual interface (`intf_name`, by custom).
+4.4.4. The component defines a string configuration field that is used to look up the virtual interface (`if_name`, by custom).
 
 4.4.5. The vkit defines a scope name under which all virtual interfaces for all instances of the verification kit are stored.
 
-4.4.6. *Recommended*: Use the macro `` `cmn_get_intf `` to pull virtual interface handles from the resource database.
+4.4.6. *Recommended*: Use the macro `` `cmn_get_if `` to pull virtual interface handles from the resource database.
```

Comment:
I didnt see any use of the macro cmn_get_intf when running the grep command on the core-v-verif repo, so I assume it is okey to just change the macro name? Not sure though, as the macro is probably defined somewhere?

Reply:
We have not really discussed these macros.  The names used in the guidelines are as they were from the original contribution, and they really should be reviewed.  For now, the change you have made is OK.